data_steward/cdr_cleaner/clean_cdr.py

```diff
@@ -179,7 +180,8 @@
     (CleanHeightAndWeight,),  # dependent on MeasurementRecordsSuppression
     (UnitNormalization,),  # dependent on CleanHeightAndWeight
     (DropZeroConceptIDs,),
-    (CleanMappingExtTables,)
+    (CleanMappingExtTables,),
+    (CreatePersonExtTable,)
```

Comment:
`(CleanMappingExtTables,),` needs to be the last rule run in this list.  It's purpose is to drop any mapping records that no longer have a record in the OMOP tables.

Reply:
will adjust the order!